stdlib/public/core/Integers.swift.gyb

```diff
@@ -2740,8 +2797,18 @@ ${unsafeOperationComment(x.operator)}
 ${operatorComment('&' + x.operator, True)}
   @_inlineable // FIXME(sil-serialize-all)
   @_transparent
-  public static func &${x.operator} (lhs: Self, rhs: Self) -> Self {
-    return lhs.${x.name}ReportingOverflow(${callLabel}rhs).partialValue
```

Comment:
These implementations can be left alone. No need to have the compiler do more work than is necessary to arrive at the same result.